--- zope_interface/setup.py
+++ b/setup_new.py
@@ -122,16 +122,14 @@ setup(name='zope.interface',
         "Framework :: Zope3",
         "Topic :: Software Development :: Libraries :: Python Modules",
       ],
-      packages=find_packages('src'),
+      packages=['zope', 'zope.interface'],
       package_dir={'': 'src'},
       namespace_packages=["zope"],
       cmdclass={
           'build_ext': optional_build_ext,
       },
-      test_suite='zope.interface.tests',
       include_package_data=True,
       zip_safe=False,
-      tests_require=tests_require,
       install_requires=['setuptools'],
       extras_require={
           'docs': ['Sphinx', 'repoze.sphinx.autointerface'],
